packages/contracts-bedrock/test/libraries/rlp/RLPReader.t.sol

```diff
@@ -19,26 +20,31 @@ contract RLPReader_readBytes_Test is Test {
         assertEq(RLPReader.readBytes(hex"7f"), hex"7f");
     }
 
+    /// forge-config: default.allow_internal_expect_revert = true
```

Comment:
If the diff is manageable, I would suggest we move to the workaround that doesn't need this override, since it's safer long term: https://github.com/ethereum-optimism/superchain-ops/pull/618#discussion_r1959876152. Let me know what you think